code/modules/vore/fluffstuff/custom_items_vr.dm

```diff
@@ -1495,4 +1495,15 @@
 	desc = "A well-worn slime custom-made yellow plushie, extensively hugged and loved. It reeks of lemon."
 	icon = 'icons/vore/custom_items_vr.dmi'
 	icon_state = "lemonplush"
-	attack_verb = list("blorbled", "slimed", "absorbed", "glomped")
\ No newline at end of file
+	attack_verb = list("blorbled", "slimed", "absorbed", "glomped")
+	
+//Bricker98:Nettie Stough
+/obj/item/modular_computer/tablet/preset/custom_loadout/elite/nettie
```

Comment:
This should be a subtype of custom tablet with the icon changed rather than a subtype of the elite tablet. Might fix some of the issues you described as well.